docs/source/release_notes.rst

```diff
@@ -11,6 +11,17 @@ Release Notes
 
 .. Thanks to the following people for contributing to this release:
 
+**Future Release**
+    * Enhancements
+    * Fixes
+        * Use deploy keys for dependency check (:pr:`1245`:)
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+Thanks to the following people for contributing to this release:
```

Comment:
A commented out `Future Release` section is already here, we don't need to add another section